### PR TITLE
Add workarea to list of production flux users

### PR DESCRIPTION
**Pull request by mcfearsome**

### CODE REVIEW

**Comment by 2opremio** [comment]
Thanks! 🌷

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ container images and config changes are propagated to the cluster.
 |[Under Armour](https://www.underarmour.com)|[VSHN](https://vshn.ch)|[Walmart Chile](https://www.walmartchile.cl)
 |[Weave Cloud](https://cloud.weave.works)|[Workable](https://www.workable.com)|[Working Group Two](https://wgtwo.com)
 |[Yusofleet](https://yusofleet.com)|[Yad2](https://yad2.co.il)|[Zaaksysteem](https://zaaksysteem.nl)
-|||
+|[Workarea](https://www.workarea.com)||
```
Comment (2opremio) [inline README.md]: Can you keep the alphabetical order?

Comment (mcfearsome) [reply README.md]: Didn't even notice that, sorry, will do

Comment (2opremio) [reply README.md]: Thanks!

**Pull request merged**